dadi/frontend/components/FieldImage/FieldImageRoutes.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+
+const config = require(paths.config)
+const AWS = require('aws-sdk')
+
+const ImageFieldRoute = function (app) {
+
+  if (config.get('FieldImage.s3.enabled')) {
+    this.initAWS()
+    this.S3 = this.initS3()
+
+    app.post({
+      name: 'images3sign', // This allows us to reuse the auth request
+      path: '/fields/image/s3/sign'
+    }, 
+    (req, res, next) => {
+      res.header('Content-Type', 'application/json')
+      let response = {route: req.route.name}
+
+      if (req.isAuthenticated()) {
+        if (req.params && req.params.fileName) {
+
```

Comment:
The idea was to separate the return from the rest of the block. In this case, the return is the only element in the block so I'd say there's no need to add a space. Otherwise, we'd have to add it in places like [L46](https://github.com/dadi/publish/blob/8181d8fca9a4e32cd8c06acb97dcc9120a0505e1/dadi/frontend/components/FieldImage/FieldImageRoutes.js#L46) and [L48](https://github.com/dadi/publish/blob/8181d8fca9a4e32cd8c06acb97dcc9120a0505e1/dadi/frontend/components/FieldImage/FieldImageRoutes.js#L48).